cal/api/serializers.py

```diff
@@ -59,14 +60,15 @@ def get_fields(self, *args, **kwargs):
 
     class Meta:
         model = ColorCategory
-        fields = ('color_index', 'label', 'hours', 'calendar')
+        fields = ('id', 'color_index', 'label', 'hours', 'calendar')
 
     def get_hours(self, obj):
-        calendar_ids = self.context['calendar_ids']
-        start = self.context['start']
-        end = self.context['end']
+        calendar_ids = self.context.get('calendar_ids')
+        start = self.context.get('start')
+        end = self.context.get('end')
```

Comment:
@MaxFangX Copied `get_serializer_context` from `api/views.py` into `ColorCategoryDetail` as requested. However when I test it, it gets this error: `super(type, obj): obj must be an instance or subtype of type`, any idea why?